app/src/main/java/com/android/partagix/ui/components/ItemUi.kt

```diff
@@ -103,27 +103,19 @@ fun ItemUi(
     },
     index: Int = 0,
 ) {
-  val date: Date =
-      if (loan.startDate.before(Date())) {
-        loan.endDate
-      } else {
-        loan.startDate
-      }
   var expanded by remember { mutableStateOf(expandState) }
-  val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy")
-
-  val available = true // TODO
   var availability = "Now available"
-
   if (!available) {
-    availability = "Unavailable"
+    availability = "unavailable now"
   }
-
-  if (isLender) {
+  if (isBorrower && loan.state == LoanState.ACCEPTED) {
     availability = "Accepted but not started"
   }
-  if (isLender && (loan.state == LoanState.ONGOING)) { // normal case of borrowed item
-    availability = ""
+  if (isBorrower && loan.state == LoanState.ONGOING) { // normal case of borrowed item
+    availability = "Available"
+  }
+  if (isOwner && !available) { // normal case of borrowed item
+    availability = "borrowed"
```

Comment:
This text is displayed to the user. It should be properly formatted
```suggestion
    availability = "Borrowed"
```